<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Category</class>
 <widget class="QMainWindow" name="Category">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Category Layout</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
        QWidget
        {
            color: #fff;
            background-color: #01889C;
            selection-background-color:#3d8ec9;
            selection-color: black;
            background-clip: border;
            border-image: none;
            outline: 0;
        }

        QPushButton
        {
            color: silver;
            background-color: #484846;
            border-width: 2px;
            border-color: #4A4949;
            border-style: solid;
            padding-top: 2px;
            padding-bottom: 2px;
            padding-left: 10px;
            padding-right: 10px;
            border-radius: 4px;
        }

        QPushButton:focus {
            background-color: #302F2F;
            color: white;
        }

        QPushButton:checked{
            background-color: #302F2F;
            border-color: #6A6969;
        }

        QPushButton:hover {
            border: 2px solid #fff;
            color: silver;
        }

        QPushButton:pressed
        {
            background-color: #484846;
        }

    </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>701</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic;&quot;&gt;Select One Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fruits">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Fruits</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>strawberry.png</normaloff>strawberry.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_animals">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Animals</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>cat.png</normaloff>cat.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sports">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Sports</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ball.png</normaloff>ball.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
